src/pytrajplot/plotting/analyse_trajectories.py

```diff
@@ -82,6 +82,7 @@ def _get_traj_dict(data, number_of_trajectories=None, traj_length=None) -> dict:
 
 def _check_dateline_crossing(
     lon: pd.Series,
+    number_of_times: int,
```

Comment:
can you document the new parameter?

Reply:
notice also the doc of the return type is wrong.

Reply:
Done.